hs_core/templates/search/search.html

```diff
@@ -0,0 +1,141 @@
+<form method="get" action=".">
```

Comment:
Where is this file ever actually used? I don't see any other references to it in the branch.